packages/oruga-next/src/components/select/Select.vue

```diff
@@ -183,6 +184,10 @@ const props = defineProps({
         type: [String, Array, Function] as PropType<ComponentClass>,
         default: undefined,
     },
+    variantClassOnRoot: {
```

Comment:
Why not just add another variant class like `rootVariantClass`? This tweak seems unnecessarily complex.

Reply:
fixed